Guides/Anomalies/Anomalies & You - Introduction and Notes

```diff
@@ -0,0 +1,53 @@
+░█░░█░ [head=3]Anomalies & You[/head] 
+░▄▀▀▄░ [head=3]Written by Researcher Pip Soak[/head] 
+[/color] 
+────────────────────────────────────────── 
+[head=2]Shift Start Recommendations[/head]
+
+CORDS ON - Remember, we’re not Service Crew, we're important.
+
+[head=3]Essential Materials:[/head]
+Gather rods (made from steel) and glass to create at least 4 reinforced glass panels for anomaly containment.
+
+Split the 30 plasma in the Anomalies room, using 15 plasma to create 4 Reinforced Plasma Glass panels. These are more durable and useful for containing radiation.
+
+Radiation Suit:
+Wear your (usually provided) NanoTrasen Radiation Suit for protection against high levels of radiation during prolonged exposure.
+
+Tool Kit:
+Bring a full toolbelt and don’t forget a welding mask. When out on the "field" you will have to be self-sufficient. 
+
+Anomaly Scanner:
+If short-staffed, wait until the Anomaly scanner is unlocked. Ideally, always carry an anomaly scanner.
+
+────────────────────────────────────────── 
+[head=2]Anomaly Scanning[/head]
+
+Once an anomaly is scanned, the scanner UI shows four particle types with their effects:
+
+Danger Particles: Increase the anomaly’s severity; use carefully to raise point output, but beware of safety risks.
+Unstable Particles: Lower the anomaly's stability; generally avoided by conscientious scientists.
+Containment Particles: Raise anomaly stability at the cost of its health, ideal for elimination.
+Transform Particles: Alters anomaly modifications, impacting point multipliers between -50% to 190%. Avoid gambling on transformations as risks are high.
+
+[head=3]After Scanning:[/head]
+
+Return the scanner to Science and connect it to an Anomaly Vessel.
+Label and organize your equipment by type and location, e.g., “[bold]FIRE ANOM - (LOCATION)[/bold].” Communicate findings with the RD or Science team.
+
+────────────────────────────────────────── 
+
+[head=2]Points and Severity Tips[/head]
+
+This is all subjective to individual experience levels and crew expectations.
+
+My personal suggestion is to not go over 75% severity, as if you fall unconcious and have no labled the contained Anom's scanner, other staff may not know where/what to do for the location and particle type to contain. Having 25% wiggle room allows for mistakes.
+
+For high output of points, i would suggest aiming for a transformation modification that is low risk, such as visible spectrum (invisibility) that will increase the point output above 150%.
+
+Whilst i do not reccomend this, upon first creation and low severity of the anom, if you are experienced and have a timer of 3+ mins for the next pulse, you can push the severity to 60%+ and then use "Unstable Particles" to create a danger level, this can give you a huge boost (150+) points for a short duration, then once you are close to 30 seconds left until next pulse, return it to stable. DO NOT REPEAT THIS STEP, as higher severity with have shorter times between pulses. 

```

Comment:
```suggestion
Whilst I do not recommend this, upon first creation and low severity of the anom, if you are experienced and have a timer of 3+ mins for the next pulse, you can push the severity to 60%+ and then use "Unstable Particles" to create a danger level, this can give you a huge boost (150+) points for a short duration, then once you are close to 30 seconds left until next pulse, return it to stable. DO NOT REPEAT THIS STEP, as higher severity with have shorter times between pulses. 
```